Corzbank.Services/GenericService.cs

```diff
@@ -115,5 +115,11 @@ public async Task<T> FindByCondition(Expression<Func<T, bool>> expression)
 
 			return result;
         }
+
+		public IEnumerable<T> GetByCondition(Expression<Func<T, bool>> expression)
+		{
+			var result = _dbContext.Set<T>().Where(expression).AsQueryable().ToList();
```

Comment:
Why did you add .AsQueryable and then used .toList?

Reply:
https://stackoverflow.com/a/17996264

Reply:
resolved